ssv/spectest/tests/valcheck/test.go

```diff
@@ -62,6 +62,10 @@ func (test *SpecTest) valCheckF(signer types.BeaconSigner) qbft.ProposedValueChe
 	}
 }
 
+func (tests *SpecTest) GetPostState() (interface{}, error) {
+	return nil, nil
```

Comment:
these tests don't have any post state?

Reply:
this specifically no

There are some places though that we return nil even though there is a post state.
But in those places we can implement this function as we touch the tests later.

Personally for me, this PR is more about getting the infra right

Reply:
This specifically no.
For others we will add later as needed.